shenyu-admin/src/main/java/org/apache/shenyu/admin/service/impl/AppAuthServiceImpl.java

```diff
@@ -414,9 +405,9 @@ public List<AppAuthData> listAll() {
             return new ArrayList<>();
         }
 
-        Set<String> idSet = appAuthDOList.stream().map(BaseDO::getId).collect(Collectors.toSet());
-        Map<String, List<AuthParamData>> paramMap = this.prepareAuthParamData(idSet);
-        Map<String, List<AuthPathData>> pathMap = this.prepareAuthPathData(idSet);
+        List<String> idList = appAuthDOList.stream().map(BaseDO::getId).collect(Collectors.toList());
```

Comment:
maybe also add `.distinct()`?

Reply:
unnecessary, same result.

Reply:
`SELECT * FROM app_auth WHERE id IN(1518171137579126784)` is better than `SELECT * FROM app_auth WHERE id IN(1518171137579126784,...(50000 repetitions),1518171137579126784)` when executing SQL.

Reply:
`EXPLAIN SELECT * FROM app_auth WHERE id IN(1518171137579126783, 1518171137579126784)`
`EXPLAIN SELECT * FROM app_auth WHERE id IN(1518171137579126783,...(50000 repetitions),1518171137579126784)`
Try it again.

Reply:
I've tried to execute SQL directly. The SQL that is always shorter executes faster

Reply:
Changed.